src/main/java/com/rultor/agents/daemons/StartsDaemon.java

```diff
@@ -69,6 +69,10 @@
 @ToString
 @EqualsAndHashCode(callSuper = false)
 public final class StartsDaemon extends AbstractAgent {
+    /**
+     * The default image for the project.
+     */
+    private static final String IMAGE = "yegor256/rultor";
```

Comment:
@sebing this is kind of code duplication, we need to extract this constant to separate class